test/tests/security/test_api_security_login.py

```diff
@@ -0,0 +1,120 @@
+'''
+Copyright 2015, DellEMC LLC
+
+   ScriptName: test_api_security_login.py
+       Author: Torrey Cuthbert
+        Email: torrey.cuthbert@dell.com
+  Create Date: 04/09/2017
+
+      Purpose: This script consists of cases intended to test RackHD login functions.
+               As of date, there are three user account roles (Administrator, Operator, and ReadOnly).
+               We will first test the RackHD endpoint's ability to create an account for each of these
+               roles for both RackHD and Redfish interfaces (redfish=False).  We will then test the
+               Redfish endpoint's ability to also create Redfish and RackHD accounts (Redfish=True).
+               In both cases, all user roles will have established accounts in both endpoints and
+               will possess tokens for role base access and permission.
+
+           EX: python run_tests.py -stack 2 -config dellemc-test/config-mn/ -test tests/security/test_api_security_login.py
+               python run_tests.py -stack 2 -test tests/security/test_api_security_login.py
+'''
+
+import sys
+import flogging
+import fit_common
+import exrex
+from nose.plugins.attrib import attr
+sys.path.append(fit_common.TEST_PATH + "/classes")
```

Comment:
This should be addressed in import of fit_path.  if not, we need to update it.  Check the template.
import fit_path # NOQA: unused import